invite/settings.py

```diff
@@ -36,4 +36,4 @@ def get_service_name():
     return getattr(
         settings,
         'INVITE_SERVICE_NAME',
-        Site.objects.get_current().name)
+        Site.objects.get_current(request=request).name)
```

Comment:
Hmmm, I don't think that would work because of [this line](https://github.com/django/django/blob/stable/1.6.x/django/contrib/sites/models.py#L102). The Sites framework would be installed, so it would just try to use get_current() again, which would fail when the SITE_ID isn't defined in the settings. We might have to reduce backwards compatibility for this update to 1.8+